app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/MapFieldValueToUser.kt

```diff
@@ -0,0 +1,56 @@
+package org.dhis2.usescases.datasets.dataSetTable.dataSetSection
+
+import org.dhis2.Bindings.toDate
+import org.dhis2.R
+import org.dhis2.commons.resources.ResourceManager
+import org.dhis2.data.forms.dataentry.tablefields.FieldViewModel
+import org.dhis2.utils.DateUtils
+import org.hisp.dhis.android.core.common.ValueType
+import org.hisp.dhis.android.core.dataelement.DataElement
+
+class MapFieldValueToUser(
+    val resources: ResourceManager,
+    val repository: DataValueRepository) {
+
+    fun map(field: FieldViewModel, dataElement: DataElement): String? {
```

Comment:
This class looks like a Use Case, isn't it?. why don't you use[ invoke operator](https://developer.android.com/topic/architecture/domain-layer#use-cases-kotlin) if it only has a public method?